packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -40,59 +44,29 @@ function getBlockDisplayName( blockType, attributes ) {
 	return formatlessDisplayName;
 }
 
-export default function BlockNavigationList( {
-	blocks,
-	selectedBlockClientId,
-	selectBlock,
-	showAppender,
-
-	// Internal use only.
-	showNestedBlocks,
-	parentBlockClientId,
-} ) {
-	const shouldShowAppender = showAppender && !! parentBlockClientId;
+function NavigationList( { blocks, selectBlock, selectedBlockClientId, showAppender, showBlockMovers, showNestedBlocks, parentBlockClientId } ) {
+	const isTreeRoot = ! parentBlockClientId;
 
 	return (
-		/*
-		 * Disable reason: The `list` ARIA role is redundant but
-		 * Safari+VoiceOver won't announce the list otherwise.
-		 */
-		/* eslint-disable jsx-a11y/no-redundant-roles */
-		<ul className="editor-block-navigation__list block-editor-block-navigation__list" role="list">
-			{ map( blocks, ( block ) => {
-				const blockType = getBlockType( block.name );
-				const isSelected = block.clientId === selectedBlockClientId;
-
+		<ul className="editor-block-navigation__list block-editor-block-navigation__list" role={ isTreeRoot ? 'tree' : 'group' }>
```

Comment:
Is it correct that, if `isTreeRoot` is false, the `<ul>` will have `role="group"` and the `<li>`s will have `role="treeitem"`?

Reply:
Yeah, the inner `ul`s should have be groups, if you scroll down a bit here there's some example markup:
https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html